.teamcity/src/main/kotlin/common/Os.kt

```diff
@@ -18,10 +18,27 @@ package common
 
 private const val killAllGradleProcessesUnixLike = """
 free -m
-ps aux | egrep 'Gradle(Daemon|Worker)'
-ps aux | egrep 'Gradle(Daemon|Worker)' | awk '{print ${'$'}2}' | xargs kill
-sleep 10
-ps aux | egrep 'Gradle(Daemon|Worker)'
+ps aux | egrep 'Gradle(Daemon|Worker)' | awk '{print ${'$'}2}'
+
+COUNTER=0
+
+while [ ${'$'}COUNTER -lt 10 ]
```

Comment:
We can increase the wait time now to 30 seconds, given that we are using polling.

Reply:
```suggestion
while [ ${'$'}COUNTER -lt 30 ]
```